docs/container.md

```diff
@@ -0,0 +1,69 @@
+<!--
+SPDX-FileCopyrightText: Red Hat, Inc.
+SPDX-License-Identifier: GPL-2.0-or-later
+-->
+
+# Containerized imageio server
```

Comment:
Maybe: "Using the ovirt-imageio container"?

I think the focus of this document is how people can use ovirt-imageio 
without oVirt. This can be used for testing (e.g. test code interacting
with ovirt-imageio) or for tranferring images on other systems.